specs/binding-kafka.spec/src/main/java/io/aklivity/zilla/specs/binding/kafka/internal/KafkaFunctions.java

```diff
@@ -1639,6 +1641,13 @@ public KafkaFetchFlushExBuilder fetch()
             return new KafkaFetchFlushExBuilder();
         }
 
+        public KafkaProduceFlushExBuilder produce()
+        {
+            flushExRW.kind(KafkaApi.PRODUCE.value());
+
+            return new KafkaProduceFlushExBuilder();
+        }
+
```

Comment:
Please make sure we have coverage in `KafkaFunctionsTest` for these new methods.

Reply:
I've already added these:
shouldMatchProduceFlushExtension, shouldMatchProduceFlushExtensionWithLatestOffset, shouldGenerateProduceFlushExtension, shouldGenerateProduceFlushExtensionWithLatestOffset

Reply:
I've already added these:
shouldMatchProduceFlushExtension, shouldMatchProduceFlushExtensionWithLatestOffset, shouldGenerateProduceFlushExtension, shouldGenerateProduceFlushExtensionWithLatestOffset